_spinnaker_install_admin_guides/install-on-kubernetes.md

```diff
@@ -0,0 +1,601 @@
+---
+layout: post
+title: Installing Spinnaker on Kubernetes
+order: 20
+published: true
+redirect_from:
+  - /spinnaker_install_admin_guides/install_on_kubernetes/
+  - /spinnaker_install_admin_guides/install-on-kubernetes/
+  - /spinnaker-install-admin-guides/install_on_kubernetes/
+---
+
+This guide describes how to install Spinnaker in Kubernetes.
+
+It uses an AWS S3 bucket for persistent storage of Spinnaker Applications and Pipelines; this can be swapped out for an equivalent blob storage mechanism such as Minio, GCS, or AZS.
+
+This document currently does not fully cover the following (see [Next Steps](#next-steps) for some links to achieve these)
+
+* TLS Encryption
+* Authentication/Authorization
+* Add K8s accounts to deploy to
+* Add cloud accounts to deploy to
+
+Note: This document is focused on Armory Spinnaker, but can be adapted to install Open Source Spinnaker by using a different Halyard container and a corresponding different Spinnaker version
```

Comment:
... by using the [OSS Halyard container](gcr.io/spinnaker-marketplace/halyard:stable) and a corresponding [OSS Spinnaker version](https://www.spinnaker.io/community/releases/versions/)